anise/src/astro/orbit_geodetic.rs

```diff
@@ -190,66 +189,62 @@ impl CartesianState {
         Ok(self.apoapsis_km()? - self.frame.mean_equatorial_radius_km()?)
     }
 
+    /// Returns the geodetic latitude, geodetic longitude, and geodetic height, respectively in degrees, degrees, and kilometers.
+    ///
+    /// # Algorithm
+    /// This uses the Heikkinen procedure, which is not iterative. The results match Vallado and GMAT.
+    ///
+    pub fn latlongalt(&self) -> PhysicsResult<(f64, f64, f64)> {
+        let a = self.frame.mean_equatorial_radius_km()?;
```

Comment:
Add units to these variables, add permalink to the algorithm on Wikipedia, and consider updating Wikipedia with units and additional explanation.

Reply:
The equations are not homogeneous.